miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/UploadController.java

```diff
@@ -141,15 +145,13 @@ public String uploadProjectSampleDeliveryForm(MultipartHttpServletRequest reques
       throw new IOException("Cannot upload file - project does not exist");
     }
 
-    boolean taxonCheck = (Boolean) request.getSession().getServletContext().getAttribute("taxonLookupEnabled");
-
     try {
       for (MultipartFile fileItem : getMultipartFiles(request)) {
         uploadFile(Project.class, projectId, fileItem);
         File f = filesManager.getFile(Project.class, projectId, fileItem.getOriginalFilename().replaceAll("\\s+", "_"));
         List<Sample> samples = FormUtils.importSampleDeliveryForm(f);
         log.info("Importing samples from form: " + samples.toString());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Format specifiers should be used instead of string concatenation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3457)